docs/webpack-dev-server.config.js

```diff
@@ -18,7 +18,12 @@ const config = {
       //material-ui requires will be searched in src folder, not in node_modules
       'material-ui/lib': path.resolve(__dirname, '../src'),
       'material-ui': path.resolve(__dirname, '../src'),
+      'material-ui-docs': path.resolve(__dirname, 'src/app'),
```

Comment:
Looks like are doing it only for the `MobileTearSheet`. Should we kill this component :grin:?